generate/generate_file_test.go

```diff
@@ -256,77 +256,6 @@ func TestGenerateFile(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
I moved tests related to subdir to a single place instead of testing on both genfile/genhcl. Seemed less error prone on this case.